dts/bindings/timer/ti,am654-dmtimer.yaml

```diff
@@ -0,0 +1,22 @@
+# Copyright (C) 2023 BeagleBoard.org Foundation
+# Copyright (C) 2023 S Prashanth
+#
+# SPDX-License-Identifier: Apache-2.0
+
+description: TI dual-mode Timer
+
+compatible: "ti,am654-dmtimer"
+
+include: base.yaml
+
+properties:
+  reg-shift:
+    type: int
+    required: true
+    description: quantity to shift the register offsets by
+
```

Comment:
What is the significance of this property and how is it used? Do we have different versions of the IP with different shift offsets like some 8250 UART has? Also not seeing the property here in the Linux bindings : https://git.kernel.org/pub/scm/linux/kernel/git/next/linux-next.git/tree/Documentation/devicetree/bindings/timer/ti,timer-dm.yaml , since the bindings are present upstream can we try to closely match them here for the supported features?

Reply:
Seems to be something left over, will remove.